CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 - Fix sigsegv when no plugin_feed_info.inc file present. [#278](https://github.com/greenbone/gvm-libs/pull/278)
+- Fix missing linking to libgnutls in util/CMakeLists.txt. [#291](https://github.com/greenbone/gvm-libs/pull/291)
```

Comment:
```suggestion
- Fix missing linking to libgnutls in util/CMakeLists.txt. [#290](https://github.com/greenbone/gvm-libs/pull/290)
```

Reply:
jjnicola pointed out that this is in the 11.0.1 section of the changelog.  Doesn't refer to this exact PR but close enough, so closing.